_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,290 @@
+
+The "RFC" (request for comments) process is intended to provide a consistent
+path and expectation for a few things specifically: tooling changes, process changes, or new standards
+that might modify the acceptance criteria for new packages entering core-plans. The purpose
+is to ensure that all project stakeholders can be confident and aware of the direction core-plans
+and their maintenance are heading in.
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make any change considered "substantial"
+to the Core-Plans repository, tooling, or maintenance processes (including the RFC process itself).
+What constitutes a "substantial" change may evolve over time based on the community,
+but may initially include the following.
+
+  - Any process or operational changes to the Core-Plans Maintenance Policies
+  - Any package feature or rewrite that breaks backwards compatibility for current users.
+  - Any Removal of a package's features (see breaking change).
+  - Changes to the standardized operational tooling in use by Core-Plans maintainers.
+  - Any changes to the standard core-plans accepted style and baseline.
+  - Any inbound Habitat change that obligates all entries in Core-Plans to change.
+
+Some changes do not require an RFC:
+
+  - Package refactors/rephrases, changes that are transparent to user.
+  - Adding net "New" software to the core repository.
+  - Additions only likely to be _noticed by_ other maintainers.
+    invisible to target package users.
+  - Bug fixes for currently available packages.
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other package developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over in the  #core-plans channel in slack as well as filing and discussing ideas on the
+[Core-Plans issue tracker](https://github.com/habitat-sh/core-plans/issues)
+
+As a rule of thumb, receiving encouraging feedback from long-standing package
```

Comment:
I recall learning that "rule of thumb" is a poor choice of phrase due to the origins. How about "As a general rule," ?